portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,52 @@
 <!DOCTYPE html>
-<html>
-  <head>
+<html lang="en">
+
+<head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <link rel="stylesheet" href="home-styles.css">
+    <title>Page Name</title>
+</head>
+
+<body>
+    <div class="content-container">
+        <div class="header-container">
+            <h1 class="title">Page Title</h1>
+            <p class="description">Subheading giving information about the page!</p>
+        </div>
+        
+        <form method="POST" action="/form-handler">
+            <div class="search-container">
+            <!--Maybe implement autofill?-->
+            <input type="text" name="search-val" value="" placeholder="Search for a topic...">
+            </div>
+        </form>
+        <div class="topic-container">
+            <div class="topic-card">
+                <h2>Topic 1 title</h2>
+                <div class="topic-img-container"><img src="images/topic-1.png" alt="topic 1 image" ></div>
+                
+                <p>Topic description. Here, we can put some info about the topic, but this isn't a topic, it's a picture of casca. If you ask me, that's far better :D</p>
+                <a class="topic-btn" href="">LEARN MORE</a>
+            </div>
+
+
+
+            <div class="topic-card">
+                <h2>Topic 2 title</h2>
+                <div class="topic-img-container"><img src="images/topic-2.jpg" alt="topic 2 image" ></div>
+                
+                <p>Topic description. Here, we can put some info about the topic, but this isn't a topic, it's a picture of angi from fire punch</p>
+                <a class="topic-btn" href="">LEARN MORE</a>
+            </div>
+
+            <div class="topic-card">
+                <h2>Topic 3 title</h2>
+                <div class="topic-img-container"><img src="images/topic-3.png" alt="topic 3 image" ></div>
+                
+                <p>Topic description. Here, we can put some info about the topic, but this isn't a topic, it's a picture of c.c. from code geass  </p>
+                <a class="topic-btn" href="">LEARN MORE</a>
+            </div>
+        </div>
     </div>
-  </body>
-</html>
+    
```

Comment:
LGTM!! All it's well, but maybe u can add a footer with about us that contains our names and the goal of the project :D

Reply:
yeah I think a footer is a great addition actually, we could also include any sort of resources we use in there as well